deploy/kubernetes/README-CN.md

```diff
@@ -0,0 +1,111 @@
+NAME                               READY   STATUS    RESTARTS   AGE
+nebula-exporter-5964b765c9-4xfkm   1/1     Running   0          44s
+nebula-exporter-5964b765c9-t26n2   1/1     Running   0          83s
+```
+
+_访问 [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) 来查看命令文档。_
+
+### 使用 helm 卸载
+
+```shell
+$ helm uninstall nebula-exporter --namespace="${NEBULA_EXPORTER_NAMESPACE}"
+```
+
+_访问 [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) 来查看命令文档。_
+
+## 可选: chart 参数
+
+参考 [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing).
+要查看带有详细注释的所有可配置选项，请访问  [values.yaml](https://github.com/vesoft-inc/nebula-stats-exporter/blob/master/charts/nebula-exporter/values.yaml),
+或者运行如下命令：
+
+```shell
+$ helm show values nebula-exporter/nebula-exporter
+```
+
+下表列出了图表的可配置参数及其默认值。
+
+| 参数 | 描述 | 默认值 |
+|:---------|:-----------|:-------|
+| `nameOverride` | 重写 chart 的 name | `""` |
+| `fullnameOverride` | 重写 chart 的 full name | `""` |
+| `cluster` | 收集哪个 Nebula Cluster 的 metrics ，默认是同一个 namespace 下的所有集群 | `""` |
+| `replicaCount` | Nebula stats exporter 的 replica 数 | `1` |
+| `startUp.listenPort` | Nebula metrics 服务监听端口 | `9100` |
+| `startUp.maxRequests` | 最大并行抓取请求数，使用 0 则不限制 | `40` |
+| `image.repository` | Nebula stats exporter image repository | `vesoft/nebula-stats-exporter` |
+| `image.tag` | Nebula stats exporter image tag | `v0.0.4` |
+| `image.pullPolicy` | Nebula stats exporter imagePullPolicy | `IfNotPresent` |
+| `serviceAccount.create` | 指定是否应创建 service account | `true` |
+| `serviceAccount.annotations` | 添加到 service account 的 annotations | `{}` |
+| `serviceAccount.name` | 要使用的 service account 的名称。 如果未设置且 create 为 true，则使用 fullname 模板生成名称 | `nebula-exporter` |
+| `serviceMonitor.enabled` | 是否开启 serviceMonitor | `false` |
+| `serviceMonitor.prometheusServiceMatchLabels` | Prometheus 的 ServiceMatchLabels, 使用 `kubectl -n `${namespace}` get prometheus `${name}` -o=jsonpath='{.spec.serviceMonitorSelector.matchLabels}'` 获取。| `{}` |
+| `ingress.enabled` | 是否开启 ingress | `false` |
+| `ingress.annotations` | Ingress 的 annotations | `{}` |
+| `ingress.hosts` | Ingress 的 hosts | `chart-example.local` |
+| `ingress.paths` | Ingress 的 paths | `["/metrics"]` |
+| `ingress.tls` | Ingress 的 tls | `[]` |
+| `ingress.ingressClass` | Ingress 的 ingressClass | `""` |
+| `podAnnotations` | Nebula stats exporter pod annotations | `{}` |
+| `podLabels` | Nebula stats exporter pod labels | `{}` |
+| `podLabels` | Nebula stats exporter resources | `{}` |
+| `livenessProbe` | Nebula stats exporter livenessProbe | `{"failureThreshold":2,"httpGet":{"path":"/health","scheme":"HTTP"},"initialDelaySeconds":30,"timeoutSeconds":10}` |
+| `nodeSelector` | Nebula stats exporter nodeSelector | `{}` |
+| `tolerations` | Nebula stats exporter tolerations | `{}` |
+| `affinity` | Nebula stats exporter affinity | `{}` |
```

Comment:
Add more description relate k8s to make it clearly?

Reply:
Those who use operator should know the probabilities related to kubernetes. Do we need to reiterate it?